diem-unified-charter.md

```diff
@@ -18,11 +18,12 @@ The DIEM WG will define an architecture and discovery mechanism enabling digital
 
 # Architectural Consideration
 
-"To bear an emblem" means to present or display an identifying mark. 
+"To *bear* an emblem" means to present or display an identifying mark.
 The entity that bears the emblem is respectively the bearer or emblem holder. 
 This is often a separate entity from the creator or original designer of the emblem.
 
-"To validate an emblem" means to confirm the authenticity or legitimacy of a particular symbol or design, often by checking its details against a known standard or reference point. 
+"To *validate* an emblem" means to confirm the authenticity or legitimacy of a particular symbol or design, often by checking its details against a known standard or reference point.
+In particular, one can validate an emblem's *binding*, that is, whether the emblem was intended to be presented by its bearer.
```

Comment:
```suggestion
"To *verify* an emblem's binding" means to ensure that the bearer has not forged, stolen, or tampered with the attributes of the emblem.
```

Reply:
To be honest, I am not sure if the binding is from the emblem to the protocol, or from the emblem to identifiers... it seems you could have both... a binding to the protocol, and a binding to the attributes that the emblem is intending to mark.

Reply:
You can bind to many things, that is correct. I don't think this suggestion improves the charter because now "validating binding" seems redundant to "validating an emblem." I think, binding ended up in this charter because I stressed multiple times that binding works differently for digital and physical assets. (How do I know this DE is supposed to go on this wooden crate? Much more difficult than this web server.)

Why don't we take an alternative approach and remove "binding" completely and only talk about validation? I will open a separate PR.

Reply:
I left related comments here: https://github.com/OR13/diem-draft-charter/issues/30#issuecomment-2694575370

I think teasing apart "protocol binding" from "marked property" binding will help avoid some of the confusion.